From 8098e6e16c1b7a403c27744508c9892d482061fa Mon Sep 17 00:00:00 2001
From: Watson Sato <wsato@redhat.com>
Date: Thu, 14 Sep 2017 19:07:46 +0200
Subject: [PATCH] Drop firewalld default zone and sshd port fixes

Providing a fix for 'firewalld_sshd_port_enabled' can be very complicated
and will very likely not fit to everyone's use case. And because of that
we drop remediation for 'set_firewalld_sshd_port', which is causing the
remediated machine to refuse all connections.
---
 shared/templates/static/bash/set_firewalld_default_zone.sh      | 10 ----
 1 file changed, 10 deletions(-)
 delete mode 100644 shared/templates/static/bash/set_firewalld_default_zone.sh

diff --git a/shared/templates/static/bash/set_firewalld_default_zone.sh b/shared/templates/static/bash/set_firewalld_default_zone.sh
deleted file mode 100644
index ada8b68a7..000000000
--- a/shared/templates/static/bash/set_firewalld_default_zone.sh
+++ /dev/null
@@ -1,6 +0,0 @@
-# platform = Red Hat Enterprise Linux 7
-grep -q ^DefaultZone= /etc/firewalld/firewalld.conf && \
-  sed -i "s/DefaultZone=.*/DefaultZone=drop/g" /etc/firewalld/firewalld.conf
-if ! [ $? -eq 0 ]; then
-    echo "DefaultZone=drop" >> /etc/firewalld/firewalld.conf
-fi
